src/Identifier/TokenIdentifier.php

```diff
@@ -41,13 +41,13 @@ class TokenIdentifier extends AbstractIdentifier
      */
     public function identify(array $data)
     {
-        $dataField = $this->getConfig('dataField');
-        if (!isset($data[$dataField])) {
+        $tokenField = $this->getConfig('tokenField');
+        if (!isset($data[$tokenField])) {
             return null;
         }
 
         $conditions = [
-            $this->getConfig('tokenField') => $data[$dataField],
+            $this->getConfig('dataField') => $data[$tokenField],
```

Comment:
you are right. I came out from this issue https://github.com/cakephp/authentication/issues/353#issuecomment-590667892 I think data and token name is confusing. I expected that data is db field and token for token. But its other way arround. And also when you use custom token key name, here https://github.com/cakephp/authentication/pull/376/files/90a2daea03f470ef1294d9ac423bd42870a17ece#diff-b7a10d2c99be4b6528582b9471a0fb86R92 was fixed to 'sub' key name and because of that custom naming did not work and I did some debug. But now I see that naming is vice vera as I thought. But this https://github.com/cakephp/authentication/blob/master/src/Authenticator/JwtAuthenticator.php#L92 is still the issue.

Reply:
`JwtAuthenticator` should use `JwtSubjectIdentifier`, you don't need to mess around with the `TokenIdentifier`.

Reply:
@GrandFelix Switching the use of config options would be a backwards compatibility break. Since the issue is about making use of `IdentifierInterface::CREDENTIAL_JWT_SUBJECT` only `JwtAuthenticator` needs changes.

Reply:
yes... will fix that. thanks.